invenio_app_ils/records/permissions.py

```diff
@@ -21,26 +26,28 @@ class RecordPermission(Permission):
     - Delete access to admin and specified users.
     """
 
-    create_actions = ['create']
-    read_actions = ['read']
-    update_actions = ['update']
-    delete_actions = ['delete']
+    _actions = {'POST': 'create',
```

Comment:
I would start writing the object properties on new line like you do for `NEW_ITEM` in `test_item_permissions_endpoint.py`
```
_actions = {
  'POST': 'create,
...
}
```